rxkeystore/src/main/java/com/nexenio/rxkeystore/provider/cipher/symmetric/aes/AesCipherProvider.java

```diff
@@ -43,6 +43,11 @@ public AesCipherProvider(RxKeyStore rxKeyStore) {
                 });
     }
 
+    @Override
+    public Single<SecretKey> generateKey(@NonNull String alias, @Nullable Integer keyPurposes, @NonNull Context context) {
+        return generateKey(alias, context);
```

Comment:
The generation logic should be implemented in this method and probably make use of the keyPurposes. `generateKey(@NonNull String alias, @NonNull Context context)` should call this method

Reply:
This can't be used in the default key generators and the ones used by us as it is not implemented.